### PR TITLE
chore: bump SDK version to `0.46.13`

### PR DESCRIPTION
**Pull request by lumtis**
# Description

- Bump Cosmos SDK to `0.46.13`
- Replace Tendermint with CometBFT as per https://github.com/cosmos/cosmos-sdk/releases/tag/v0.46.13
- Update Tendermint dependencies
- Bump Ethermint to `0.22.0` (necessary for CometBFT upgrade)

close https://github.com/zeta-chain/node/issues/500

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Include instructions and any relevant details so others can reproduce. 

- [ ] Tested CCTX in localnet
- [ ] Tested in development environment
- [ ] Go unit tests
- [ ] Go integration tests
- [ ] Tested via GitHub Actions 

# Checklist:

- [ ] I have added unit tests that prove my fix feature works

### CODE REVIEW

**Comment by lumtis** [comment]
Putting it ready for review, tough we may want to release `0.7.0` before merging it?

File: go.mod [hunk 1]
```diff
@@ -29,15 +29,15 @@ require (
 
 require (
 	cosmossdk.io/errors v1.0.0-beta.7
-	cosmossdk.io/math v1.0.0-beta.4
+	cosmossdk.io/math v1.0.0-rc.0
 	github.com/99designs/keyring v1.2.1
 	github.com/binance-chain/tss-lib v1.3.2
-	github.com/btcsuite/btcd v0.22.2
+	github.com/btcsuite/btcd v0.23.4
```
Comment (kingpinXD) [inline go.mod:35]: @brewmaster012 Can you verify this change will not impact in any way

**Review by kingpinXD — APPROVED** [review]
LGTM, 
let's wait from @brewmaster012 approval as well before merging

File: go.mod (hunk 1 above)
Comment (brewmaster012) [reply go.mod:35]: actually the version of btcd is pinned down to 
`github.com/btcsuite/btcd => github.com/btcsuite/btcd v0.22.1`
by the replace directive

**Comment by brewmaster012** [comment]
@lumtis great PR
a few questions before merge:
1. does this upgrade require any kind of migration to be deployed on a3 testnet? 

if no then let's merge

**Comment by lumtis** [comment]
>does this upgrade require any kind of migration to be deployed on a3 testnet?

No, the release doesn't have breaking changes.

**Comment by brewmaster012** [comment]
now that we are at it, perhaps update a few vulnerable dependencies as well
btcd 0.22.2 => 0.22.3
github.com/hashicorp/go-getter v1.6.1 => v1.7.0
github.com/ipld/go-ipld-prime v0.9.0 => v0.19.0?

**Comment by lumtis** [comment]
> now that we are at it, perhaps update a few vulnerable dependencies as well btcd 0.22.2 => 0.22.3 github.com/hashicorp/go-getter v1.6.1 => v1.7.0 github.com/ipld/go-ipld-prime v0.9.0 => v0.19.0?

Updated for the last two packages https://github.com/zeta-chain/node/pull/889/commits/1c7a021395d7df3fe02b5b89eeebb6f4b4e317e6

Regarding `btcd `, shouldn't we remove the `replace` statement and use `v0.23.4`? https://pkg.go.dev/vuln/GO-2022-1098

**Comment by lumtis** [comment]
Can't upgrade `btcd` to `0.23.4` because smoke tests use `btcd/btcec` which no longer exist (or been moved?) in the newer version. Maybe we could handle this upgrade in a separate PR? @brewmaster012

**Comment by brewmaster012** [comment]
> Can't upgrade `btcd` to `0.23.4` because smoke tests use `btcd/btcec` which no longer exist (or been moved?) in the newer version. Maybe we could handle this upgrade in a separate PR? @brewmaster012

Sounds good. It's not an easy upgrade because the version also needs to be paired with Bitcoin node (bitcoin core) version. But we can probably pinned down to btcd v0.22.3 which fixes a vuln

**Comment by lumtis** [comment]
> > Can't upgrade `btcd` to `0.23.4` because smoke tests use `btcd/btcec` which no longer exist (or been moved?) in the newer version. Maybe we could handle this upgrade in a separate PR? @brewmaster012
> 
> Sounds good. It's not an easy upgrade because the version also needs to be paired with Bitcoin node (bitcoin core) version. But we can probably pinned down to btcd v0.22.3 which fixes a vuln

Issue of `0.23`: https://github.com/zeta-chain/node/issues/922

Commit for `0.22.3`: https://github.com/zeta-chain/node/pull/889/commits/257c025193d7fb7013ddfda13c92d2341f6c3aee

**Review by brewmaster012 — APPROVED** [review]

**Pull request merged**